maikthomas/sframe
# Add decryptionRestored and decryptFailed events

add `decryptFailed` and `decryptionRestored ` events

Test instructions:
You can checkout locally and run 
`npm run example`

Change the passwords for sender and receiver and observe the events in the console
